### PR TITLE
Allow running executables with runtime params

### PR DESCRIPTION
**Pull request by dkarlovi**
This follows #21, that one should be merged first.

### CODE REVIEW

File: src/ProcessManagerBundle/Controller/ExecutableController.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function runAction(Request $request)
             ]);
         }
 
-        $this->get('process_manager.registry.processes')->get($exe->getType())->run($exe);
+        $this->get('process_manager.registry.processes')->get($exe->getType())->run($exe, $request->request->all());
```
Comment (dpfaffenbauer) [inline src/ProcessManagerBundle/Controller/ExecutableController.php]: I think a form to pre-process the data would be better here -> so we only get validated data. WDYT?

Comment (dkarlovi) [reply src/ProcessManagerBundle/Controller/ExecutableController.php]: Definitely agree. :+1:

Comment (dpfaffenbauer) [reply src/ProcessManagerBundle/Controller/ExecutableController.php]: done

**Comment by dkarlovi** [comment]
@dpfaffenbauer what's this waiting for?

**Pull request merged**